pkg/asset/internal/templates.go

```diff
@@ -198,6 +198,43 @@ spec:
         hostPath:
           path: /var/lock
 `)
+
+	KencTemplate = []byte(`apiVersion: "extensions/v1beta1"
+kind: DaemonSet
+metadata:
+  name: kenc
+  namespace: kube-system
+  labels:
+    k8s-app: kenc
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: kenc
+      annotations:
+        checkpointer.alpha.coreos.com/checkpoint: "true"
+    spec:
+      nodeSelector:
+        master: "true"
+      hostNetwork: true
+      containers:
+      - image: gcr.io/coreos-k8s-scale-testing/kenc
+        name: kenc
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /etc/kubernetes/selfhosted-etcd
```

Comment:
For pod checkpointing we use:

`/etc/kubernetes/checkpoint-configmaps`
`/etc/kubernetes/checkpoint-secrets`

Maybe for consistency we use `/etc/kubernetes/checkpoint-iptables`?